packages/strapi-design-system/src/Field/FieldLabel.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import { Box } from '../Box';
+import { Flex } from '../Flex';
 import { useField } from './FieldContext';
 import { Typography } from '../Typography';
 
 const TypographyAsterisk = styled(Typography)`
   line-height: 0;
 `;
 
-export const FieldLabel = ({ children, required, ...props }) => {
+const Action = styled(Box)`
```

Comment:
Proposal from my comment posted before: 
`const Action = styled(Flex)`
That would fix the alignment issue

Reply:
That's a great proposal. I keep forgetting that `styled` can be used that way. Thanks.

Added in d707436e291900c0d4aefb947f2ae7d7b3bd9fa4